app/helpers/language_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module LanguageHelper
+  class UkrLanguageHelper
```

Comment:
no need to name a class helper. it's already declared in module LanguageHelper. change to UkrLaunguage

Reply:
fixed with commit "changed name of helper class to UkrLanguage"